### PR TITLE
OSDOCS-9322: Rewriting the steps to modify a compute machine set

### PR DESCRIPTION
**Pull request by jeana-redhat**
Version(s):
4.11+

Issue:
[OSDOCS-9322](https://issues.redhat.com//browse/OSDOCS-9322)

Link to docs preview:
[Modifying a compute machine set](https://70199--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/modifying-machineset.html#machineset-modifying_modifying-machineset)

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/machineset-modifying.adoc [hunk 1]
```diff
@@ -5,91 +5,144 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="machineset-modifying_{context}"]
-= Modifying a compute machine set
+= Modifying a compute machine set by using the CLI
 
-To make changes to a compute machine set, edit the `MachineSet` YAML. Then, remove all machines associated with the compute machine set by deleting each machine or scaling down the compute machine set to `0` replicas. Then, scale the replicas back to the desired number. Changes you make to a compute machine set do not affect existing machines.
+Changes that you make to a compute machine set do not affect existing machines.
+To create machines that reflect the updated configuration in the `MachineSet` custom resource (CR), you must scale the compute machine set.
```
Comment (elmiko) [inline modules/machineset-modifying.adoc:11]: i wonder if this is clear enough, would it be worthwhile to state that only new machines created after the configuration change will take the updated configuration?

**Review by elmiko — COMMENTED** [review]
looks mostly good to me, just a question

File: modules/machineset-modifying.adoc (hunk 1 above)
Comment (jeana-redhat) [reply modules/machineset-modifying.adoc]: Thanks - hopefully the new commit helps :crossed_fingers:

**Comment by sunzhaohua2** [comment]
LGTM

**Review by elmiko — COMMENTED** [review]
update lgtm, thanks!

**Comment by jab-rh** [comment]
@jeana-redhat, LGTM, thanks!

**Pull request merged**